Makefile

```diff
@@ -0,0 +1,85 @@
+VERSION ?= 0.0.0
+GOOS ?= darwin
+GOARCH ?= amd64
```

Comment:
Can we have these default values for the environment running `make`, by using `uname` or something? (I'm using Ubuntu for my daily work, so I don't want to specify GOOS=linux everytime :P)

Reply:
Thanks for the comment. I use `go env` to retrieve these variables, instead of hardcoding.  17ebc5a